src/get-started/java-binding.md

```diff
@@ -712,6 +712,47 @@ service_name1 = "/path/to/service1_artifact.jar"
 service_name2 = "/path/to/service2_artifact.jar"
 ```
 
+## Logging Configuration
+
+Java Binding uses two different methods for logging - [`log4j`][log4j-home]
+in Java modules and [`env_logger`][env_logger-home] in Rust modules.
+
+### Rust Logging
+
+Rust logs are produced by Exonum Core and can be used to monitor the
+status of the blockchain node, including information about the block
+commitment and the consensus status.
+
+Rust logs are disabled by default and controlled by the `RUST_LOG`
+environment variable. It is recommended to set `info` logging level:
+
+```bash
+export RUST_LOG=warn,exonum=info,exonum-java=info
+```
+
+Log entries go to `stderr` by default.
+
+See [`env_logger` documentation][env_logger-docs] for more information
+on possible configuration options.
+
+### Java Logging
+
+Java logs are produced by Java Binding internal code and can also be
+used to monitor Java services internal logic.
+
+Java logging configuration is controlled by configuration file. Example
```

Comment:
Java logging configuration is controlled by the configuration file specified by the `ejb-log-config-path` parameter.  If no file was provided, the logs are disabled. 
Exonum Java package provides an example `log4j-fallback.xml` configuration file that can be found at the installation directory.
Also, see [Application Guide][app-tutorial] for more information on
configuring the Exonum Java App.